### PR TITLE
Remove OpenFunction(FAAS layer) relative UI templates and menu item.

### LINKED ISSUE #11569: [Feature] Plan to remove or deprecate OpenFunction monitoring from SkyWalking
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

The OpenFunction UI integration part has lacked maintenance for years. With the newly introduced MQE, this part is hard to follow up.
By following our principle, it is time to remove it as no committer is maintaining it and keeping it updated anymore.

FYI @apache/skywalking-committers especially @arugal as previous owner. @smartboy37597 has tried to fix it, but still fails even in installation as the showcase also lacked maintenance.

@kezhenxu94 Please remove it from the showcase, as we are not capable of maintaining it anymore.

At the main repository, I am going to wait for @smartboy37597 's last try, if still fails in the week, I would like to remove the menu and UI templates directly. The codes don't need to change, as OpenFunction is still based on go2sky only.

BTW, go2sky(not ASF project) is being abandoned as well, this makes me believe the integration can't last long enough.

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wu-sheng**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #11569.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

FYI @smartboy37597 This will be merged if your side is still blocked.

@kezhenxu94 Please remove ofn from showcase no matter this gets merged or not

### CODE REVIEW

**Review by wankai123 — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]

**Comment by wu-sheng** [comment]
@wankai123 @kezhenxu94 Let's remove this. The last fix attempt fails.

**Review by kezhenxu94 — APPROVED** [review]
